collection-scripts/gather_metallb_logs

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 BASE_COLLECTION_PATH="must-gather"
-METALLB_NS=($(oc get subs -A --field-selector=metadata.name=metallb-operator -o jsonpath='{.items[*].metadata.namespace}'))
+METALLB_NS=$(oc get subs -A -o template --template  '{{range .items}}{{if eq .spec.name "metallb-operator"}}{{.metadata.namespace}}{{end}}{{end}}')
```

Comment:
there is an extra space after `--template`

Reply:
fixed the spacing. and you're right, with double quotes it's better, added.